### PR TITLE
Minor revision to bikes edit

### PR DESCRIPTION
**Pull request by Robertgervais**

### CODE REVIEW

**Comment by Robertgervais** [comment]
Image 5-10-19 at 10 00 PM

**Comment by Robertgervais** [comment]
Image 5-14-19 at 4 29 PM

File: app/views/admin/bikes/_bike.html.haml [hunk 1]
```diff
@@ -19,18 +22,22 @@
...
-              %td
-                %span.convertTime.preciseTime
-                  = l bike.current_stolen_record.date_stolen, format: :convert_time
-            %tr
-              %td
-                From
-              %td
-                = bike.short_address(bike.current_stolen_record)
+    - if bike.stolen and bike.current_stolen_record.present?
+      .col-lg-6.border.border-danger.mt-auto
```
Comment (sethherr) [inline app/views/admin/bikes/_bike.html.haml]: Nice. I like only specifying col width lg here. 👍

Comment (sethherr) [reply app/views/admin/bikes/_bike.html.haml]: Also, thanks for explaining what `mt-auto` does to me 🙂

**Pull request merged**